docs/events.md

```diff
@@ -0,0 +1,88 @@
+ import {HTML5_EVENTS as Html5Events, CUSTOM_EVENTS as CustomEvents} from 'path/to/../event/events'
+```
+
+
+
+## HTML5 events list
+| Event Name | Event String | Description |
+|--|--|--|
+| ABORT | abort | Fires when the loading of an audio/video is aborted
+| CAN_PLAY | canplay | Fires when the browser can start playing the audio/video
+|CAN_PLAY_THROUGH | canplaythrough | Fires when the browser can play through the audio/video without stopping for buffering|
+|DURATION_CHANGE | durationchange| Fires when the duration of the audio/video is changed|
+| EMPTIED|emptied | Fires when the current playlist is empty|
+|ENDED | ended| Fires when the current playlist is ended|
+| ERROR| error| Fires when an error occurred during the loading of an audio/video|
+| LOADED_DATA| loadeddata | Fires when the browser has loaded the current frame of the audio/video
+|  LOADED_METADATA| loadedmetadata |Fires when the browser has loaded meta data for the audio/video
+|  LOAD_START| loadstart|Fires when the browser starts looking for the audio/video
+|  PAUSE| pause |Fires when the audio/video has been paused
+|PLAY  | play |Fires when the audio/video has been started or is no longer paused
+|  PLAYING| playing |Fires when the audio/video is playing after having been paused or stopped for buffering
+| PROGRESS | progress|Fires when the browser is downloading the audio/video
+| RATE_CHANGE | ratechange |Fires when the playing speed of the audio/video is changed
+| SEEKED | seeked |Fires when the user is finished moving/skipping to a new position in the audio/video
+|  SEEKING| seeking |Fires when the user starts moving/skipping to a new position in the audio/video
+| STALLED | stalled |Fires when the browser is trying to get media data, but data is not available
+| SUSPEND |  suspend| Fires when the browser is intentionally not getting media data
+| TIME_UPDATE| timeupdate |Fires when the current playback position has changed
+| VOLUME_CHANGE | volumechange |Fires when the volume has been changed
+| WAITING |  waiting| Fires when the video stops because it needs to buffer the next frame
+
+## Custom events list
+|  Event Name| Event String | Description |
+|--|--|--|
+| ENTER_FULLSCREEN | enterfullscreen| Fires when the player enters fullscreen |
+| EXIT_FULLSCREEN | exitfullscreen | Fires when the player exits fullscreen |
+| REQUESTED_ENTER_FULLSCREEN | requestedenterfullscreen | Fires when the player received a request to enter fullscreen |
+| REQUESTED_EXIT_FULLSCREEN |  requestedexitfullscreen|Fires when the player received a request to exit fullscreen  |
+| AUTOPLAY_FAILED | autoplayfailed | Fires when browser fails to autoplay with sound |
+| FALLBACK_TO_MUTED_AUTOPLAY | fallbacktomutedautoplay | Fires when browser fails to autoplay with sound but start muted autoplay instead |
+| CHANGE_SOURCE_STARTED | changesourcestarted | Fires when change source flow started |
+| CHANGE_SOURCE_ENDED |  changesourceended| Fires when change source flow ended |
```

Comment:
I think we need to clarify both this one and the "CHANGE_SOURCE_STARTED" - do explain what this means rather than just the event. WDYT?

Reply:
well, i'm debating it. it's much to explain. i think if someone want's to dig in he should go into the code. what do you think?